wiki/graphics/2d/tile-maps.md

```diff
@@ -3,7 +3,7 @@ title: Tile maps
 ---
 # Maps
 
-libGDX features a generic maps API. All map related classes can be found in the [com.badlogic.gdx.maps](https://libgdx.badlogicgames.com/nightlies/docs/api/index.html?com/badlogic/gdx/maps/package-use.html) [(code)](https://github.com/libgdx/libgdx/tree/master/gdx/src/com/badlogic/gdx/maps) package. The root package contains the base classes, sub-packages contain specialized implementations for tile maps and other forms of maps.
+libGDX features a generic maps API. All map related classes can be found in the [com.badlogic.gdx.maps](https://libgdx.badlogicgames.com/ci/nightlies/docs/api/com/badlogic/gdx/maps/package-use.html) [(code)](https://github.com/libgdx/libgdx/tree/master/gdx/src/com/badlogic/gdx/maps) package. The root package contains the base classes, sub-packages contain specialized implementations for tile maps and other forms of maps.
```

Comment:
This one is intentionally left at libgdx.badlogicgames.com due to no package use pages at javadocs.io or in our Maven JARs. Lyze is investigating.

Reply:
Probably solved in https://github.com/libgdx/libgdx/pull/6877. But there's no point changing that link until 1.11.1 is out, as we'd just have a broken link until then.

Reply:
Okay, so we'll have to keep that in mind when 1.11.1 releases.